docs/epub-press.yaml

```diff
@@ -0,0 +1,214 @@
+      requestBody:
+        description: ebook request body
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Book'
+      responses:
+        '200':
+          description: create book response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  id: 
+                    description: book id
+                    type: string
+        '400':
+          description: bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+                
+  /books/:id/status:
+    get:
+      summary: Get the book publishing status.
+      description: Get the book publishing status
+      tags: 
+        - books
+      parameters:
+        - name: id
+          in: query
+          schema:
+            type: string
+      responses:
+        '200':
+          description: book publishing status
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                  progress:
+                    type: number
+                    
+  /books/:id/download:
+    get:
+      summary: Download the book.
+      description: Download the book.
+      tags: 
+        - books
+      parameters:
+        - name: id
+          in: query
+          schema:
+            type: string
+      responses:
+        '200':
+          description: the book bits
+          content:
+            application/epub+zip:
+              schema:
+                type: string
+                format: binary
+                
+  /books/:id/email:
+    get:
+      summary: Email the book.
+      description: Email the book.
+      tags:
+        - books
+      parameters:
+        - name: id
+          in: query
+          schema:
+            type: string
+        - name: email
+          in: query
+          schema:
+            type: string
+      responses:
+        '200':
+          description: TODO
+          content:
+             text/html:
+              schema:
+                type: string
+    
+    
+components:
+  schemas:
+    ClientVersion:
+      properties:
+        minCompatible:
+          type: string
+        message:
+          type: string
+    Section:
+      properties:
+        content:
+          description: HTML content of the section
+          type: string
+        title: 
+          description: Plain text title
+          type: string
+    Error:
+      properties:
+        errors:
+          type: array
+          items:
+            type: object
+            properties:
+              status: 
+                type: string
+                description: http status code
+              detail:
+                type: string
+                description: description of error
```

Comment:
It looks like some descriptions have capitals and periods and others are all lowercase? 
Consistency could be nice (the descriptions with capitals and periods look more like descriptions).